src/components/MenuItem/MenuItem.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { hotPrice } from "../../data"
+
+export class MenuItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bread: "Standard",
+      isHot: false,
+      total: this.props.price
+    };
+
+    this.action = this.action.bind(this);
+    this.changeBread = this.changeBread.bind(this);
+    this.changeHot = this.changeHot.bind(this);
+  }
+
+  changeBread(event) {
+    const { breads, price } = this.props;
+    const bread = breads[event.target.value].name;
+    const breadPrice = breads[event.target.value].price;
+    const total = !this.state.isHot ? price + breadPrice : (price + breadPrice) + hotPrice;
```

Comment:
`price + breadPrice` is used twice so could go into its own variable? `sandwichPrice`? Also the statement after the `:` has unnecessary brackets man 👍